ci/container-images/permaplant-mdbook/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM rust:1.67.1-slim
 
-RUN apt update && apt install -y libpq-dev
-RUN cargo install mdbook mdbook-mermaid
+RUN apt update && apt install -y libpq-dev curl pkg-config ca-certificates build-essential make perl gcc libc6-dev
+RUN cargo install mdbook mdbook-mermaid --verbose
```

Comment:
Is this needed for devs?

Reply:
curl pkg-config ca-certificates build-essential make perl gcc libc6-dev This was needed to perform the linkchecking. The --verbose not.